bin/check-rds-events.rb

```diff
@@ -117,7 +117,7 @@ def maint_clusters
         # fetch the last 15 minutes of events for each cluster
         # that way, we're only spammed with persistent notifications that we'd care about.
         clusters.each do |cluster_name|
-          events_record = rds.describe_events(start_time: (Time.now - 900).iso8601, source_type: 'db-instance', source_identifier: cluster_name)
+          events_record = rds.describe_events(start_time: (Time.now - 900).strftime('%Y-%m-%dT%H:%M:%SZ'), source_type: 'db-instance', source_identifier: cluster_name)
```

Comment:
I suspect this is not actually what you want. Rather, encode the current UTC time in ISO-8601 format:

```ruby
(Time.now - 900).getgm.iso8601
# => "2018-04-12T16:26:36Z"
```